<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Traffic Sign Recognition</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QListWidget" name="listWidget">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>637</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionLanguage"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuVisitImages">
    <property name="title">
     <string>Browse Images</string>
    </property>
    <addaction name="actionOpenSingleImage"/>
    <addaction name="separator"/>
    <addaction name="actionBrowseTestSet"/>
    <addaction name="actionNextImage"/>
    <addaction name="actionPreviousImage"/>
    <addaction name="actionAnimation"/>
    <addaction name="actionFirstImage"/>
    <addaction name="actionLastImage"/>
    <addaction name="actionMLPDynamicTraining"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionTestCanny"/>
   </widget>
   <widget class="QMenu" name="menuWorkFlow">
    <property name="title">
     <string>WorkFlow</string>
    </property>
    <addaction name="actionSetTrainData"/>
    <addaction name="actionSetTestData"/>
    <addaction name="separator"/>
    <addaction name="actionMLPTraining"/>
    <addaction name="actionMLPTestingForSingleImage"/>
    <addaction name="actionMLPTestingForTestData"/>
    <addaction name="separator"/>
    <addaction name="actionSVMTraining"/>
    <addaction name="actionSVMTestingForSingleImage"/>
   </widget>
   <addaction name="menuVisitImages"/>
   <addaction name="menuWorkFlow"/>
   <addaction name="menu"/>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="navigatorToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFirstImage"/>
   <addaction name="actionPreviousImage"/>
   <addaction name="actionAnimation"/>
   <addaction name="actionNextImage"/>
   <addaction name="actionLastImage"/>
   <addaction name="actionMLPDynamicTraining"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSettings"/>
  </widget>
  <action name="actionLanguage">
   <property name="text">
    <string>Language</string>
   </property>
  </action>
  <action name="actionBrowseTestSet">
   <property name="text">
    <string>Load Test Image Sequences </string>
   </property>
   <property name="toolTip">
    <string>Load Test Image Sequences </string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/resource/settings.png</normaloff>
     <normalon>:/image/resource/settings.png</normalon>:/image/resource/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionNextImage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/resource/next.png</normaloff>:/image/resource/next.png</iconset>
   </property>
   <property name="text">
    <string>Next Image</string>
   </property>
  </action>
  <action name="actionPreviousImage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/resource/previos.png</normaloff>:/image/resource/previos.png</iconset>
   </property>
   <property name="text">
    <string>Previous Image</string>
   </property>
   <property name="toolTip">
    <string>PreviousImage</string>
   </property>
  </action>
  <action name="actionAnimation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <selectedoff>:/image/resource/play.png</selectedoff>
     <selectedon>:/image/resource/pause.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Animation</string>
   </property>
  </action>
  <action name="actionFirstImage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/resource/first.png</normaloff>:/image/resource/first.png</iconset>
   </property>
   <property name="text">
    <string>First Image</string>
   </property>
  </action>
  <action name="actionLastImage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/resource/last.png</normaloff>:/image/resource/last.png</iconset>
   </property>
   <property name="text">
    <string>Last Image</string>
   </property>
   <property name="toolTip">
    <string>Last Image</string>
   </property>
  </action>
  <action name="actionTestCanny">
   <property name="text">
    <string>Test Canny</string>
   </property>
  </action>
  <action name="actionMLPTraining">
   <property name="text">
    <string>MLP Training</string>
   </property>
  </action>
  <action name="actionSetTrainData">
   <property name="text">
    <string>Set Train Data</string>
   </property>
   <property name="toolTip">
    <string>Set Train Data</string>
   </property>
  </action>
  <action name="actionSVMTraining">
   <property name="text">
    <string>SVM Training</string>
   </property>
  </action>
  <action name="actionOpenSingleImage">
   <property name="text">
    <string>Open Single Image</string>
   </property>
  </action>
  <action name="actionSVMTestingForSingleImage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>SVM Testing For Single Image</string>
   </property>
  </action>
  <action name="actionMLPTestingForSingleImage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>MLP Testing For Single Image</string>
   </property>
  </action>
  <action name="actionSetTestData">
   <property name="text">
    <string>Set Test Data</string>
   </property>
  </action>
  <action name="actionMLPTestingForTestData">
   <property name="text">
    <string>MLP Testing For TestData</string>
   </property>
  </action>
  <action name="actionMLPDynamicTraining">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MLP Dynamic Training</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
